perl/bin/cgpVaf.pl

```diff
@@ -213,7 +203,7 @@ sub option_builder {
     }
 
     if(!defined($options{'fexc'})){
-        $options{'fexc'} = $Sanger::CGP::Vaf::VafConstants::DEFAULT_READLEN_EXCLUDE;
+        $options{'fexc'} = $Sanger::CGP::Vaf::VafConstants::DEFAULT_READS_EXCLUDE_PIELUP;
```

Comment:
This looks like a fix for incorrect filtering, how significant is the impact on on the results.  There is no commit comment that indicates why this was changed.

Reply:
typo corrected